exchanges/orderbook/calculator_test.go

```diff
@@ -468,13 +468,10 @@ func TestSimulateOrder(t *testing.T) {
 func TestGetAveragePrice(t *testing.T) {
 	var b Base
 	b.Exchange = "Binance"
-	cp, err := currency.NewPairFromString("ETH-USDT")
-	if err != nil {
-		t.Error(err)
-	}
+	cp := currency.NewBTCUSDT()
 	b.Pair = cp
 	b.Bids = []Tranche{}
-	_, err = b.GetAveragePrice(false, 5)
+	_, err := b.GetAveragePrice(false, 5)
 	if errors.Is(errNotEnoughLiquidity, err) {
```

Comment:
params need to be swapped. but while here can just assert.

Reply:
Haha yeah, I've got this fixed in #1931 
![image](https://github.com/user-attachments/assets/18722c58-8aa9-45e1-b944-8d4f48e79a9e)
so I did the bare minimum in this one, will stack this as a depends on that PR though and then just rebase it

Reply:
🚀 I will just shhh my face then